lib/mavenlink/workweek.rb

```diff
@@ -0,0 +1,4 @@
+module Mavenlink
+  class Workweek < Model
```

Comment:
I know there's not a lot to the tests for these "models", but should we add some for this and the workweek membership?

If I remember correctly, there's a shared behavior